src/app/components/pages/Market.jsx

```diff
@@ -911,7 +986,8 @@ module.exports = {
                 account: state.global.getIn(['accounts', username]),
                 history: state.market.get('history'),
                 user: username,
-                feed: state.global.get('feed_price').toJS(),
+                feed: state.global.get('feed_price'),
```

Comment:
@gl2748 i ended up changing the `open_orders_sort` prop as well as one of the other props `feed` to pass as an immutable rather than a plain object, so shouldComponentUpdate can more efficiently compare them -- unfortunately we can't define a proptypes shape for immutables -- we should eventually use a library like https://github.com/HurricaneJames/react-immutable-proptypes to do this

Reply:
ok cool.